pages/apis/rest_api/pipelines.md

```diff
@@ -898,6 +898,12 @@ Optional [request body properties](/docs/api#request-body-properties):
     <th><code>skip_queued_branch_builds_filter</code></th>
     <td>A <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch filter pattern</a> to limit which branches intermediate build skipping applies to. <p class="Docs__api-param-eg"><em>Example:</em> <code>"!main"</code><br><em>Default:</em> <code>null</code></p></td>
   </tr>
+  <tr>
+    <th><code>slug</code></th>
+    <td>A custom identifier for the pipeline. This slug will be used as the pipeline's URL path. If not provided when the pipeline name is updated, the slug will be automatically generated from the new pipeline name. If the pipeline name is not updated, the existing slug will remain unchanged.
+      <p><em>Example:</em> <code>"my-custom-pipeline-slug"</code></p>
+    </td>
```

Comment:
Suggesting wrapping in a paragraph and adding a line about the required format of slugs:
```suggestion
    <td>
      <p>A custom identifier for the pipeline. This slug will be used as the pipeline's URL path. If not provided when the pipeline name is updated, the slug will be automatically generated from the new pipeline name. If the pipeline name is not updated, the existing slug will remain unchanged.<br>
      The slug may only contain alphanumeric characters or dashes, and cannot begin with a dash.</p>
      <p><em>Example:</em> <code>"my-custom-pipeline-slug"</code></p>
    </td>
```